src/clients/c++/simple_shm_client.cc

```diff
@@ -261,22 +261,6 @@ main(int argc, char** argv)
     }
   }
 
-  // Set the context options to do batch-size 1 requests. Also request that
```

Comment:
Need to remove -I and -O option from simple_shm_client. Make this example as simple as possible

Reply:
Will remove once the output is functional before finalizing this PR